editor-components/Image_gallery/image_gallery_7/image-gallery7.tsx

```diff
@@ -0,0 +1,490 @@
+import * as React from "react";
+import { BaseImageGallery } from "../../EditorComponent";
+import styles from "./image-gallery7.module.scss";
+import { ComposerIcon } from "../../../composer-base-components/icon/icon";
+
+
+type ImageType = {
+    type: string;
+    key: string;
+    displayer: string;
+    value: string;
+  };
+  
+  
+  
+  type CardType = {
+    type: "array";
+    key: "card";
+    displayer: string;
+    value: ImageType[];
+  };
+  
+
+
+class ImageGallery7 extends BaseImageGallery {
+
+   
+    constructor(props?: any) {
+        super(props, styles);
+        
+        this.addProp({
+            type: "array",
+            key: "gallery",
+            displayer: "Gallery",
+            value: 
+                [
+                    {
+
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 1",
```

Comment:
use singular and non-contain number names.
```suggestion
                        displayer: "Card",
```